pontoon/base/static/js/manage_permissions.js

```diff
@@ -102,7 +107,7 @@ $(function() {
 
     // Update menu (must be above Copying Translators)
     $(this).addClass('hidden');
-    if ($('#project-selector .menu li:visible').length === 0) {
+    if ($('#project-selector .menu li:not(".hidden")').length === 0) {
```

Comment:
I think we're near this point when some kind if UI testing viaselenium or something similar would be helpful.